tools/1_wara_collector.ps1

```diff
@@ -272,11 +274,25 @@ $Script:Runtime = Measure-Command -Expression {
         #Set the default variable with the list of subscriptions in case no Subscription File was informed
         $Script:SubIds = Get-AzSubscription -TenantId $TenantID -WarningAction SilentlyContinue
       }
-    else
+    elseif ($PSVersionTable.OS -eq 'CBL-Mariner/Linux')
```

Comment:
What is the purpose of specifying the OS as opposed to the platform?

Reply:
The purpose here is to differentiate between Cloud Shell environment and other Unix based systems. Currently if one runs the script on mac os or linux it results in authentication error since the script expects or all Unix system an MSI. 
We can write a better way to detect Cloud Shell using $env variables for example if you like.